### PR TITLE
Implement Identity Hub Script

### PR DESCRIPTION
**Pull request by jakubbanas**

### CODE REVIEW

File: src/ad-bidders/prebid/identity-hub/identity-hub.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const logGroup = 'IdentityHub';
+
+class IdentityHub {
+	private isLoaded = false;
+	private identityHubScriptSrc = `https://ads.pubmatic.com/AdServer/js/pwt/${pubId}/${profileId}/pwt.js`;
+
+	call(): Promise<void> {
+		if (!this.isEnabled()) {
+			utils.logger(logGroup, 'disabled');
+		}
```
Comment (szwajkowski) [inline src/ad-bidders/prebid/identity-hub/identity-hub.ts:15]: i guess we should return here if it is disabled

File: src/ad-bidders/prebid/identity-hub/identity-hub.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+		utils.logger(logGroup, 'enabled');
+
+		if (!this.isLoaded) {
+			window.PWT = {
+				jsLoaded: () => {
```
Comment (szwajkowski) [inline src/ad-bidders/prebid/identity-hub/identity-hub.ts:21]: I don't get it, what's the purpose of it? It just logs that it is loaded. And does it need to be a part of `window` ?

File: src/ad-bidders/prebid/identity-hub/identity-hub.ts (hunk 1 above)
Comment (jakubbanas) [reply src/ad-bidders/prebid/identity-hub/identity-hub.ts:15]: Right, good catch!

File: src/ad-bidders/prebid/identity-hub/identity-hub.ts (hunk 2 above)
Comment (jakubbanas) [reply src/ad-bidders/prebid/identity-hub/identity-hub.ts:21]: It comes with the vendor script, after the code is executed and Identity Hub is "installed" it calls this function from the window so we know that we could proceed with GAM.

I'll put a comment there, it's confusing.

Comment (szwajkowski) [reply src/ad-bidders/prebid/identity-hub/identity-hub.ts:24]: nice, that's better

**Review by szwajkowski — APPROVED** [review]
LGTM :)

**Pull request merged**